test/integration/admin/api/api_docs_services_controller_test.rb

```diff
@@ -90,7 +90,7 @@ def test_update_invalid_params
     def test_update_unexistent_service
       put admin_api_active_doc_path(api_docs_service, format: :json), update_params(service_id: Service.last.id + 1)
       assert_response :unprocessable_entity
-      assert_contains JSON.parse(response.body).dig('errors', 'service'), 'not found'
```

Comment:
I hope that nobody has been relying on this error message format so far 👀

Reply:
Yep. It is a breaking change.

Reply:
/cc @3scale/product @3scale/qe @3scale/support 

When one tries to update an existing ActiveDoc modifying the {{service_id}} attribute to a value that is NOT a valid ID of an API of the provider, instead of the previous error response in the typical format of ActiveRecord validation errors:

```json
{ "errors" : { "service" : ["not found"] } }
```

the PR modifies the response to the standard format:
```json
{ "error" : "Service not found" }
```